src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -412,22 +414,30 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
     this.realmIds = [];
     this.zoneIds = [];
     this.evaluateMigrateAndReplicationActions();
+    this.rgwMultisiteService.restartGatewayMessage$.subscribe((value) => {
+      if (value === undefined) {
```

Comment:
what if its `null` ? do we want to proceed ?
```suggestion
      if (value) {
```

Reply:
@afreen23 , we are setting the value of `restartGatewayMessage$`, only when the multi-site wizard is completed successfully so it will contain a value only in that case otherwise it would be always `undefined`. Also since `value` is a boolean so we can not check it like `if (value)` because this way it will check if the value is `true` or `false`, but in our case we need to check if the value is defined or not and then proceed accordingly. I have added a `null` check as well as per your suggestion. Also here's the recording of how the message looked like. 

https://github.com/user-attachments/assets/5e62695c-545e-4de2-afa7-c1cfdea16b03